File: src/main/java/com/trade/securities/application/port/in/GetFscBeneficiaryCertificateUseCase.java
package com.trade.securities.application.port.in;

import java.util.Map;

public interface GetFscBeneficiaryCertificateUseCase {
    Map<String, Object> getBeneficiaryCertificates(int pageNo, int numOfRows, String basDt, String itmsNm,
            String likeItmsNm);
}

File: src/main/kotlin/com/trade/securities/application/port/in/GetFscBeneficiaryCertificateUseCase.kt
package com.trade.securities.application.port.`in`

interface GetFscBeneficiaryCertificateUseCase {
    fun getBeneficiaryCertificates(pageNo: Int, numOfRows: Int, basDt: String, itmsNm: String?, likeItmsNm: String?): Map<String, Any?>
}
